src/diffusers/utils/callbacks.py

```diff
@@ -0,0 +1,116 @@
+from functools import partial
```

Comment:
I think we should move it outside of utils folder, just directly inside `scr/diffusers`, similarly to `image_proessor.py`